uploadNeuroDB/register_processed_data.pl

```diff
@@ -276,9 +333,19 @@
 ###################################
 ##           Functions           ##
 ###################################
+
 =pod
-This function returns the sessionID based on sourceFileID.
+
+=head3 getSessionID($sourceFileID, $dbh)
+
+This function returns the sessionID based on the provided sourceFileID.
+
+INPUT: source FileID, database handle
```

Comment:
I guess if we want to be consistent with other PRs, this should be:
```perl
INPUTS:
    - source FileID.
    - database handle.
```

Or maybe this is also valid. What do you think?

This comment applies to all `INPUT` sections.

Reply:
you're absolutely right! Fixing them.